index.html

```diff
@@ -6,84 +6,131 @@
+                        <option value="biomass">Biomass</option>
+                        <option value="unsure">I'm not sure!</option>
+                    </select>
+                    <button type="submit">Submit</button>
+                </form>
+            </div>
+        </section> <!-- / end About & Energy Survey section -->
+
+        <!-- Experience Section -->
         <section id="experience">
-            <h2>Experience</h2>
-                <ul>
-                    <section class="experience-block">
-                    <li><h3>Certifications and Training</h3></li>
-                        <ul>
-                            <li><em>Javascript, HTML, and CSS (In Progress)</em>: Code the Dream, Expected Completion March 2025</li>
-                            <li><em>Renewable Energy Specialization</em>: Coursera, December 2022</li>
-                            <li><em>Fundamentals of GIS</em>: Coursera, December 2022</li>
-                        </ul>
-                    </section>
-                    <section class="experience-block">
-                    <li><h3>GIS Analyst at Pine Gate Renewables</h3></li>
-                        <ul>
-                            <li>Utilized spatial analysis techniques to assist with site selection, land acquisition, and project development, contributing to informed decision-making and project success.</li>
-                            <li> Analyzed environmental and geographic datasets to identify viable project sites for utility-scale solar projects and determine potential obstacles.</li>
-                            <li> Created project maps tailored to community engagement, facilitating decision-making processes and fostering local stakeholder involvement in renewable energy initiatives.</li>
-                            <li>Built preliminary models of solar farms using Plant Predict software, ensuring accurate data visualization of all environmental factors.</li>
-                        </ul>
-                    </section>
-                    <section class="experience-block">
-                    <li><h3>Natural Resources Technician at the North Carolina Museum of Natural Sciences</h3></li>
-                        <ul>
-                            <li>Collected and processed spatial data using GPS and GIS technologies to create interactive maps, ensuring spatial accuracy of all data.</li>
-                            <li>Designed maps for both visitors and educators; included park resources and environmental features, increasing ease of access</li>
-                            <li> Conducted research on local and state renewable energy policies and regulations; synthesized data and prepared feasibility reports for renewable energy projects.</li>
-                        </ul>
-                    </section>
-                </ul>
+        <h2>Experience</h2>
+            <div class="experience-container">
+
+                <div class="experience-block">
```

Comment:
I can't quite pinpoint the issue, but for some reason in inspect the GIS portion does not appear to be in the experience-block.   If you replace it with your following experience-block and adjust the text it seems to work.   I ran out of time to figure out what is going on.